_includes/contributor/2-1-end.md

```diff
@@ -0,0 +1,3 @@
+{:.bs-callout .bs-callout-warning}
+**v2.1 End of Contribution:**
+We are no longer accepting contributions to the v2.1 release line. Please submit contributions for the [v2.2](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html) and [v2.3](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html) release lines. We made this change to increase focus, issue investigation, and pull request approvals for v2.2 and v2.3 release lines.
```

Comment:
Maybe change tense to "We no longer accept..."

Reply:
Thanks updated, made similar change to 2.2 notice.